db/range_index.go

```diff
@@ -215,3 +215,55 @@ func (r *rangeIndex) Close() {
 	r.db = nil
 	r.bucket = nil
 }
+
```

Comment:
why we want to add this?
i mean, current tests are passing, and InMemKVStore not used in production, so we don't need to add this?

Reply:
for go test